src/test/resources/CsvParserTest/authorconfig_specialCharacter_test.csv

```diff
@@ -1,4 +1,4 @@
 Repository's Location,Branch,Author's GitHub ID,Author's Email,Author's Display Name,Author's Git Author Name,Ignore Glob List
-https://github.com/reposense/testrepo-Beta.git,master,‘Processed�‘Cooked�,,,,
-https://github.com/reposense/testrepo-Beta.git,master,(codeeong),,,,
+https://github.com/reposense/testrepo-Beta.git,master,‘Processed�‘Cooked�,,"Tay Fan Gao, Douya",,
+https://github.com/reposense/testrepo-Beta.git,master,(codeeong),abc@xyz.eu,"""""Tora, S/O,"" Doyua, T.""",,
```

Comment:
create a separate test file for this as in the event of a fail arise, it may be hard to troubleshoot the causation.

Reply:
You mean to create a seperate file just to test commas and double-quotes?

Reply:
yes, also just to add on... since your this algorithm will affect all the columns, you should include tests for multiple columns too.

Reply:
Made a new test file to cater to commas and double quotes.